x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/modal/index.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useMemo, useEffect, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { Dispatch } from 'redux';
+import styled, { css } from 'styled-components';
+import {
+  EuiModal,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiModalFooter,
+  EuiButton,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { AppAction } from '../../../../../../common/store/actions';
+import { Ecs } from '../../../../../../../common/ecs';
+import { EventFiltersForm } from '../form';
+import { useEventFiltersSelector } from '../../hooks';
+import {
+  getFormHasError,
+  isCreationInProgress,
+  isCreationSuccessful,
+} from '../../../store/selector';
+import { getInitialExceptionFromEvent } from '../../../store/utils';
+import { MODAL_TITLE, MODAL_SUBTITLE, ACTIONS_CONFIRM, ACTIONS_CANCEL } from './translations';
+
+import { EventFiltersNotification } from '../notification';
+
+export interface EventFiltersModalProps {
+  data: Ecs;
+  onCancel(): void;
+}
+
+const Modal = styled(EuiModal)`
+  ${({ theme }) => css`
+    width: ${theme.eui.euiBreakpoints.l};
+    max-width: ${theme.eui.euiBreakpoints.l};
+  `}
+`;
+
+const ModalHeader = styled(EuiModalHeader)`
+  flex-direction: column;
+  align-items: flex-start;
+`;
+
+const ModalHeaderSubtitle = styled.div`
+  ${({ theme }) => css`
+    color: ${theme.eui.euiColorMediumShade};
+  `}
+`;
+
+const ModalBodySection = styled.section`
+  ${({ theme }) => css`
+    padding: ${theme.eui.euiSizeS} ${theme.eui.euiSizeL};
+
+    &.builder-section {
```

Comment:
Please don't use global class names. You can refer to another component's css class like that instead:
```
const ModalBodySection = styled.section`
  ${({ theme }) => css`
    padding: ${theme.eui.euiSizeS} ${theme.eui.euiSizeL};
    ${ModalBodySection} {
         overflow-y: scroll;
    }
  `}
`;
```

More info on that: https://styled-components.com/docs/advanced#referring-to-other-components

Reply:
I just realised that this global class name is not necessary as it is referenced to the ModalBodySection itself. Thanks for the feedback and the link!